idd/Energy+.idd.in

```diff
@@ -483,6 +483,7 @@ PerformancePrecisionTradeoffs,
       \key Mode05
       \key Mode06
       \key Mode07
+      \key Mode08
```

Comment:
OK, so Mode06 was inserted, making a new 08 one.  Transition rules to follow I see.  :+1: